service/http.directory/src/test/webapp/test.rtf

```diff
@@ -0,0 +1,7 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1348\cocoasubrtf170
```

Comment:
Suggest just having the file say hello world or saying what tests it is used for